src/webapp/components/users-selected-modal/PasswordRequirements.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+import { PASSWORD_REQUIREMENTS } from "./passwordValidation";
+import i18n from "../../../locales";
+
+interface PasswordRequirementsProps {
+    password: string;
+}
+
+export const PasswordRequirements: React.FC<PasswordRequirementsProps> = React.memo(({ password }) => {
+    return (
+        <RequirementsContainer>
+            <RequirementsTitle>{i18n.t("Password Requirements:")}</RequirementsTitle>
+            <RequirementsList>
+                {PASSWORD_REQUIREMENTS.map(requirement => (
+                    <RequirementItem key={requirement.key} valid={requirement.validator(password)}>
+                        {requirement.text()}
+                    </RequirementItem>
+                ))}
+            </RequirementsList>
+        </RequirementsContainer>
+    );
+});
+
+const RequirementsContainer = styled.div`
+    margin-top: 8px;
```

Comment:
can we use logical properties for this? 

```css
margin-block-start: 8px;
```

https://github.com/EyeSeeTea/know-how/wiki/UI-Guide#using-css-logical-properties

Also check other css code introduced in this PR